src/main/java/com/google/api/codegen/discovery/transformer/SampleNamer.java

```diff
@@ -38,8 +38,9 @@ public String getSampleClassName(String apiTypeName) {
   /**
    * Returns the variable name of the service.
    */
-  public String getServiceVarName(String apiTypeName) {
-    return localVarName(Name.lowerCamel(Name.upperCamel(apiTypeName).toLowerCamel(), "service"));
+  public String getServiceVarName(String lowerCamelApiTypeName) {
+    return localVarName(
+        Name.lowerCamel(Name.lowerCamel(lowerCamelApiTypeName).toLowerCamel(), "service"));
```

Comment:
There is a simpler way to do this:

1. Don't bother with the whole lowerCamelApiTypeName in SampleConfig
2. Use this code:

`return localVarName(Name.upperCamel(apiTypeName, "Service"));`

Basically, each of the static creation methods of Name just need to have their arguments all be in the same casing, so you would just pass the upper camel string "Service" here.